lib/ansible/modules/commands/shell.py

```diff
@@ -69,7 +69,7 @@
 notes:
    -  If you want to execute a command securely and predictably, it may be
       better to use the M(command) module instead. Best practices when writing
-      playbooks will follow the trend of using M(command) unless M(shell) is
+      playbooks will follow the trend of using M(command) unless C(shell) is
```

Comment:
Why ? If you are reading the shell documentation, you don't expect `shell` to be a link ? But the `command` documentation is quite relevant here.